### PR TITLE
completed 01, 02 tasks

### PR DESCRIPTION
**Pull request by elenaalkoli**

### CODE REVIEW

File: hw_05/task_02.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+// 3.  Вам нужно вывести в консоль числа от 1 до 100.
+//     Если число делится без остатка на 3, то выведете в консоль “число - делится на 3”.
+//     Если число делится на 5 без остатка, то то выведете в консоль “число - делится на 5”.
+//     Если число делится и на 3 и на 5 без остатка, то то выведете в консоль “число - делится и на 3 на 5”.
+//     Число 15 делится без остатка на 3 и на 5 -- пример сообщения в консоле.
+
+for (let i = 1; i <= 100; i++) {
+  if (!(i % 3) && !(i % 5)) { // if true && true (или можно !(i % 15) сразу)
```
Comment (anatoly-karpovich) [inline hw_05/task_02.js:87]: А где деление на 3 и на 5 по очереди?)

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: hw_05/task_02.js (hunk 1 above)
Comment (elenaalkoli) [reply hw_05/task_02.js]: thanks, I've added. Also added task_03

File: hw_05/task_02.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+for (let i = 1; i <= 100; i++) {
+    if (!(i % 3)) {
+        console.log(`${i} is devided on 3 without a remainder`)
+    }
+    if (!(i % 5)) {
+        console.log(`${i} is devided on 5 without a remainder`)
+    }
+    if (!(i % 3) && !(i % 5)) { // if true && true (или можно !(i % 15) сразу)
+        console.log(`${i} is devided on 3 and 5 without a remainder`)
```
Comment (anatoly-karpovich) [inline hw_05/task_02.js:94]:
```suggestion
    if (!(i % 3) && !(i % 5)) { // if true && true (или можно !(i % 15) сразу)
        console.log(`${i} is devided on 3 and 5 without a remainder`
   } else if (!(i % 3)) {
        console.log(`${i} is devided on 3 without a remainder`)
    } else  if (!(i % 5)) {
        console.log(`${i} is devided on 5 without a remainder`)
    }
```

File: hw_05/task_03.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+// const engAlphabet = [...'abcdefghijklmnopqrstuvwxyz'];
+let result = '';
+if (!message || typeof message !== "string") {
+    console.log('Please provide a valid string')
+}
+else {
+    for (let i = 0; i < message.length; i++) {
+        let codeOfSymbol = message.charCodeAt(i) //получаю юникод каждого символа
+        codeOfSymbol = codeOfSymbol + offset
+        result += String.fromCharCode(codeOfSymbol) //обратно
```
Comment (anatoly-karpovich) [inline hw_05/task_03.js:47]: You are not handling boundary values as Z and z for positive offset and A\a for negative one

Try ZzZ for offset === 1, implementation should return AaA

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**